content/blog/multiplayer/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Building Excalidraw's P2P collaboration feature
```

Comment:
the directory should be as close to the title as possible, it's suppose to be a slug.